tests/acceptance/live-test.js

```diff
@@ -29,14 +29,5 @@ module('Acceptance | live', function (hooks) {
 
     await typeIn('[data-test-input-field]', 'sanket');
     await click('[data-test-button="live-join"]');
```

Comment:
I see series of test cases removed after the clock of event, any particular reason for the same

Reply:
After the clock, the backend API call is going. Which makes the further assertion fails. We have added TODO to add the test cases for the Service.

Reply:
Should we create a ticket to address this?

Reply:
Yes sure I will create it. Thank you.